public/javascripts/Gallery/src/displays/SeverityDisplay.js

```diff
@@ -0,0 +1,48 @@
+
+/**
+ * An object that creates a display for severity based on the Mock for Sidewalk Gallery v1.1
```

Comment:
I don't think you need to mention a mock or Gallery 1.1 in the comments; I don't think it will be relevant to people reading this in the future :wink:

Reply:
This is happening in more than one place. Note that when I make a comment about something, you should check that you aren't doing it anywhere else. Because it's kind of redundant for me to add the same comment in a bunch of places :wink:

Reply:
The only other place I could find that had something like this was in the TagDisplay class. Anywhere else I should be aware of @misaugstad ?

Reply:
That's probably good. I think I ended up fixing another one or two after this at some point, so that feels about right.